### ISSUE TITLE
Unable to change theme in Magento 2.3.0

### ISSUE THREAD

**Issue by RedAtRareCandy** [magento/magento2#19874]
### Preconditions (*)

1. Magento 2.3.0
2. PHP 7.2.13

### Steps to reproduce (*)

1. Upgrade to 2.3.0
2. Log in to Admin
3. Go to Stores > Configuration
4. Click Edit on your store

### Expected result (*)

1. Page should load and transparent layer and spinning loading icon should disappear.

### Actual result (*)

1. Transparent layer and spinning loading icon do not disappear.
2. Developer Tools console shows error: "ReferenceError: Base64 is not defined"

![base64-error](https://user-images.githubusercontent.com/10075847/50218399-c977f280-03c6-11e9-94e3-3a1d99158d87.png)

**Comment by magento-engcom-team** [comment]
Hi @RedAtRareCandy. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [ ] Summary of the issue
- [ ] Information on your environment
- [ ] Steps to reproduce
- [ ] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me $VERSION instance` 

where `$VERSION` is [version tags](https://github.com/magento/magento2/tags) (starting from 2.2.0+) or develop branches (for example: 2.3-develop). 
For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@RedAtRareCandy do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [ ] no

**Comment by RedAtRareCandy** [comment]
@magento-engcom-team give me 2.3.0 instance

**Comment by magento-engcom-team** [comment]
Hi @RedAtRareCandy. Thank you for your request. I'm working on Magento 2.3.0 instance for you

**Comment by magento-engcom-team** [comment]
Hi @RedAtRareCandy, here is your [Magento instance](https://i-19874-2-3-0.instances.magento-community.engineering).
Admin access: [https://i-19874-2-3-0.instances.magento-community.engineering/admin](https://i-19874-2-3-0.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by magento-engcom-team** [comment]
Hi @engcom-backlog-nazar. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [ ] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [ ] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [ ] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [ ] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 6. Add label `Issue: Confirmed` once verification is complete. 

- [ ] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by ghost** [comment]
Hi @RedAtRareCandy  thank you for you report, i'm not able to reproduce following steps you described on clean magento 2.3-develop instance. this may be related to your custom modules or themes.

**Comment by RedAtRareCandy** [comment]
Hi,

Turns out the Base64 error was just a symptom of not being able to upload logo or favicon on the page. This commit fixed my issue entirely. Happy days!

https://github.com/magento/magento2/pull/18851/commits/ba69bfec127bed8c39c7d9e7b73d10b54449b158

**Comment by Microtribute** [comment]
Neither the 'Save and Config' button nor the orange 'Save Config' button works for me. Nothing happens. They just does nothing. There were absolutely no JavaScript errors logged in the console. There is no way to change the theme for the storefront for me and I am lost.

I am using:
- Magento Community Edition 2.3.0
- WAMP 3.1.7 with PHP 7.1.26
- Windows Server 2019 Datacenter

**Comment by DanielRuf** [comment]
@nightlybuilt please open a new issue.
